src/api/spec/lib/feature_switch/feature_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Feature do
+  let(:user) { create(:confirmed_user, :in_beta, login: 'Tom') }
```

Comment:
Why do we have to call ```create``` and not just use ```build``` we dont need the user in the db.

Reply:
True, I was wrong in my previous comment about this.

Reply:
ok, but I prefer to see this PR merged. so lets resolve it and approve as soon as everything is green.